PlayFabClientSDK/source/PluginManager.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+
+namespace PlayFab
+{
+    public class PluginManager
```

Comment:
I'd like to at least have a basic implementation of what it means to have four clones of one plugin. We shouldn't use it yet, but the definition of how it works should be here.

Reply:
Summing up after offline conversation:
- PluginManager needs to be a factory of built-in plugins
- Both instance and singleton plugins must be supported (possibly different APIs to get those)

---
In reply to: [200496925](https://github.com/PlayFab/CSharpSDK/pull/97#discussion_r200496925) [](ancestors = 200496925)

Reply:
Update: proposed API with named plugin instances, it will give us support of "clones", more specifically, multiple instances of the same plugin.

---
In reply to: [200510206](https://github.com/PlayFab/CSharpSDK/pull/97#discussion_r200510206) [](ancestors = 200510206,200496925)